tests/functional/spec/thank_you.spec.js

```diff
@@ -49,24 +49,22 @@ describe("Thank You Default View Response Enabled", () => {
       await $(DidYouKnowPage.yes()).click();
       await click(DidYouKnowPage.submit());
       await click(ThankYouSubmitPage.submit());
-      await expect(await browser.getUrl()).to.contain(ThankYouPage.pageName);
+      await expect(browser).toHaveUrlContaining(ThankYouPage.pageName);
     });
 
     it("When I navigate to the thank you page, and I have submitted less than 40 seconds ago, Then I should see the countdown timer and option to view my answers", async () => {
-      await expect(await $(ThankYouPage.viewSubmittedGuidance()).isDisplayed()).to.be.false;
-      await expect(await $(ThankYouPage.title()).getHTML()).to.contain("Thank you for completing the Test Thank You");
-      await expect(await $(ThankYouPage.viewAnswersTitle()).getHTML()).to.contain("Get a copy of your answers");
-      await expect(await $(ThankYouPage.viewAnswersLink()).getText()).to.contain("save or print your answers");
-      await expect(await $(ThankYouPage.viewSubmittedCountdown()).getHTML()).to.contain(
-        "For security, your answers will only be available to view for another",
-      );
+      await expect(await $(ThankYouPage.viewSubmittedGuidance()).isDisplayed()).toBe(false);
+      await expect(await $(ThankYouPage.title()).getHTML()).toContain("Thank you for completing the Test Thank You");
+      await expect(await $(ThankYouPage.viewAnswersTitle()).getHTML()).toContain("Get a copy of your answers");
+      await expect(await $(ThankYouPage.viewAnswersLink()).getText()).toContain("save or print your answers");
+      await expect(await $(ThankYouPage.viewSubmittedCountdown()).getHTML()).toContain("For security, your answers will only be available to view for another");
     });
 
     it("When I navigate to the thank you page, and I have submitted more than 40 seconds ago, Then I shouldn't see the option to view my answers", async () => {
-      await expect(await $(ThankYouPage.viewSubmittedGuidance()).isDisplayed()).to.be.false;
-      await browser.pause(46000); // Waiting 40 seconds for the timeout to expire (45 minute timeout changed to 35 seconds by overriding VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS for the purpose of the functional test)
-      await expect(await $(ThankYouPage.viewSubmittedGuidance()).isDisplayed()).to.be.true;
-      await expect(await $(ThankYouPage.viewSubmittedGuidance()).getHTML()).to.contain("For security, you can no longer view or get a copy of your answers");
+      await expect(await $(ThankYouPage.viewSubmittedGuidance()).isDisplayed()).toBe(false);
+      await browser.pause(50000); // Waiting 40 seconds for the timeout to expire (45 minute timeout changed to 35 seconds by overriding VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS for the purpose of the functional test)
```

Comment:
why did the brower pause change?

Reply:
That was Pete and I testing something, I completely forgot about this, will revert